hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMBlockDeletingService.java

```diff
@@ -71,11 +76,18 @@
 
   private int blockDeleteLimitSize;
 
+  /**
+   * SCMService related variables.
+   */
+  private final Lock serviceLock = new ReentrantLock();
+  private ServiceStatus serviceStatus = ServiceStatus.PAUSING;
```

Comment:
Instead of setting default value as `ServiceStatus.PAUSING`, will it make sense to initialize this variable as RUNNING during construction?

Reply:
Actually not.  BlockDeletingService should not run on follower SCMs.

Reply:
O yes thanks!